index.html

```diff
@@ -2662,9 +2662,22 @@ <h2>WoT Thing Description</h2>
 
         <section id="thing-model">
             <h2>Thing Model</h2>
-            <section class="ednote">
-                TODO: Create text that introduces the concepts of a thing model.
-            </section>
+   
+		<p>The <a>Thing Model</a> offers the oppertunity to define a template-based model for 
+		    <a>Thing Description</a> instances. The peculiarity is that a <a>Thing Model</a> has no 
```

Comment:
I am not native speaker but I haven't seen "peculiarity" being used a lot.

maybe: (In contrast to a TD,) the characteristic of Thing Model is that it has no ..